src/main/java/com/restaurant/eatenjoy/service/RestaurantService.java

```diff
@@ -132,4 +139,22 @@ private void deleteUploadFile(FileDto fileDto) {
 		fileService.deleteFileInfo(fileDto.getId());
 	}
 
+	private boolean isDeleteUploadFile(UpdateRestaurant updateRestaurant, FileDto serverFile) {
+		boolean isDelete = false;
+
+		if (serverFile != null && updateRestaurant.getUploadFile() != null) {
+			isDelete = true;
+
+			if (serverFile.getId().equals(updateRestaurant.getUploadFile().getId())) {
+				isDelete = false;
+			}
+		}
```

Comment:
이 부분은 이런식으로 개선이 가능할 것 같은데 이런식으로 좀 더 간소화시킬 수 있지 않을까요?
길어져서 가독성이 떨어지는 부분들은 메소드로 분리해서 간소화시키고요~

```suggestion
		boolean isDelete = serverFile != null && updateRestaurant.getUploadFile() != null && !serverFile.getId().equals(updateRestaurant.getUploadFile().getId());
```